src/SpendPermissionManager.sol

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+import {ERC6492Deployer} from "./ERC6492Deployer.sol";
```

Comment:
nit: put below other imports separated by newline
https://github.com/coinbase/solidity-style-guide?tab=readme-ov-file#c-group-imports-by-external-and-local-with-a-new-line-in-between